### PR TITLE
[WIP] Add collection/action check lookups, use them for sanity testing

### PR DESCRIPTION
**Pull request by felixfontein**
Will likely not (yet) work on older Ansible/ansible-base/ansible-core versions.

### CODE REVIEW

File: plugins/lookup/check_collection.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      - For example C(community.general).
+    type: list
+    elements: str
+    required: true
+"""
+
+EXAMPLES = """
+- name: Check version of community.general
+  ansible.builtin.debug:
+    msg: "community.general version {{ lookup('felixfontein.acme.check_collection', 'community.general) }}"
```
Comment (rndmh3ro) [inline plugins/lookup/check_collection.py]:
```suggestion
    msg: "community.general version {{ lookup('felixfontein.acme.check_collection', 'community.general') }}"
```

File: plugins/lookup/check_action.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      - For example C(file) or C(community.general.cobbler_system).
+    type: list
+    elements: str
+    required: true
+"""
+
+EXAMPLES = """
+- name: Check whether community.general.cobbler_system is available.
+  ansible.builtin.debug:
+    msg: "We can use cobbler_system: {{ lookup('felixfontein.acme.check_action', 'community.general.cobbler_system) }}"
```
Comment (rndmh3ro) [inline plugins/lookup/check_action.py]:
```suggestion
    msg: "We can use cobbler_system: {{ lookup('felixfontein.acme.check_action', 'community.general.cobbler_system') }}"
```

**Comment by felixfontein** [comment]
@rndmh3ro thanks for spotting that :)

**Comment by felixfontein** [comment]
@rndmh3ro @tadeboro @briantist @agaffney thanks for reviewing and discussing this (resp. later versions)!

**Pull request merged**